src/Microsoft.AspNetCore.Http/Features/ResponseCookiesFeature.cs

```diff
@@ -27,27 +24,13 @@ public class ResponseCookiesFeature : IResponseCookiesFeature
         /// <see cref="IResponseCookiesFeature"/> and the <see cref="IHttpResponseFeature"/>.
         /// </param>
         public ResponseCookiesFeature(IFeatureCollection features)
-            : this(features, builderPool: null)
-        {
-        }
-
-        /// <summary>
-        /// Initializes a new <see cref="ResponseCookiesFeature"/> instance.
-        /// </summary>
-        /// <param name="features">
-        /// <see cref="IFeatureCollection"/> containing all defined features, including this
-        /// <see cref="IResponseCookiesFeature"/> and the <see cref="IHttpResponseFeature"/>.
-        /// </param>
-        /// <param name="builderPool">The <see cref="ObjectPool{T}"/>, if available.</param>
-        public ResponseCookiesFeature(IFeatureCollection features, ObjectPool<StringBuilder> builderPool)
```

Comment:
Restored. Do I need to add back `AppendToStringBuilder` on `SetCookieHeaderValue`?